### DISCUSSION TITLE
Rendering markdown in livewire volt-component not working

### DISCUSSION THREAD

**Discussion by stevan06v** [livewire/livewire#8163] [1. Help]
**My problem**
I need to render markdown to html in my livewire-application and I tried to use the spatie-markdown package: [<x-markdown></x-markdown>](https://github.com/spatie/laravel-markdown) but i am getting alpineJs errors:

<img width="1427" alt="Bildschirmfoto 2024-03-19 um 13 04 07" src="https://github.com/livewire/livewire/assets/91951603/d1624c41-92ad-4cb4-bead-90df7fba77cf">

**Here is my livewire-component:**

```
<?php

use Livewire\Volt\Component;

new class extends Component {
    public $markdown_content;

    public function mount($markdown)
    {
        $this->markdown_content = $markdown;
    }
}; ?>
<div>
    <x-markdown wire:ignore>{{$markdown_content}}</x-markdown>
</div>

```

**Disclaimer:**
* I required: `composer require spatie/laravel-markdown`

How do I render markdown now? Are there other libaries/solutions to achieve that?

**Comment by joshhanley** [comment]
@Stevan06v you have an empty `entangle`, see screenshot below. If you fix that you shouldn't get the console error.

<img width="537" alt="image" src="https://github.com/livewire/livewire/assets/882837/72c444eb-2b37-4938-87c1-152da2fb517e">
